lib/rubocop/config_validator.rb

```diff
@@ -80,20 +54,26 @@ def validate_section_presence(name)
 
     private
 
+    def target_ruby
+      @target_ruby ||= TargetRuby.new(@config)
+    end
```

Comment:
Is there a case where it is beneficial to lazy load this? Or can we just assign it in the initializer?

Reply:
I guess we can load this eagerly. Let me check.

Reply:
This is now assigned in the initializer. I kept a private accessor method, though.